funcx_websocket_service/tests/conftest.py

```diff
@@ -11,8 +11,10 @@
 
 @pytest.fixture(autouse=True, scope="session")
 def run_server():
+    new_env = os.environ.copy()
+    new_env["FUNCX_S3_BUCKET_NAME"] = "funcx"
```

Comment:
Consider using the pytest `monkeypatch` fixture, which provides `monkeypatch.setenv`

Reply:
Ach! Nevermind. I didn't realize that this was not a contextual fixture which sets the var for use in this process, but is using `subprocess`. I'll mark this resolved.